tickets/src/main/java/no/unit/nva/publication/ticket/create/CreateTicketHandler.java

```diff
@@ -99,33 +99,38 @@ private static URI getPublisherId(Publication publication) {
         return Optional.ofNullable(publication.getPublisher()).map(Organization::getId).orElse(null);
     }
 
-    private static boolean hasValidAccessRights(RequestInfo requestInfo) {
-        return requestInfo.userIsAuthorized(AccessRight.MANAGE_DOI);
+    private static boolean hasValidAccessRights(RequestUtils requestUtils, TicketDto ticketDto) {
+        return switch (ticketDto) {
+            case DoiRequestDto doi -> requestUtils.hasAccessRight(MANAGE_DOI);
+            case PublishingRequestDto publishing -> requestUtils.hasAccessRight(MANAGE_PUBLISHING_REQUESTS);
+            case GeneralSupportRequestDto support -> requestUtils.hasAccessRight(SUPPORT);
+            case UnpublishRequestDto unpublish -> requestUtils.hasAccessRight(MANAGE_PUBLISHING_REQUESTS);
+            case null, default -> false;
```

Comment:
Not sure about following:
`case UnpublishRequestDto unpublish -> requestUtils.hasAccessRight(MANAGE_PUBLISHING_REQUESTS);`

But without passing any access right here by simply returning true will allow anyone to create UnpublishRequest.

Reply:
I think it makes sense, but perhaps the naming is wrong as MANAGE_PUBLISHING_REQUESTS is strongly connected to a particular ticket type. Should probably be MANAGE_PUBLISHING

Reply:
I think contributors and resource owner should be able to create a unpublish request if there are no files, maybe api clients should be allowed to do this (it uses different scopes) and when curator does this, we also need to check resource owner and contributors.  something that maybe should be a part of the logic here?

Reply:
my bad, unpublish is through the update publication endpoint, not here, so not relevant. The other curator stuff still may be to loosely checked here, but this change is for the better anyway